control-plane/api-gateway/binding/validation.go

```diff
@@ -322,9 +323,18 @@ func validateJWT(gateway gwv1beta1.Gateway, listener gwv1beta1.Listener, resourc
 
 func validateCertificateRefs(gateway gwv1beta1.Gateway, refs []gwv1beta1.SecretObjectReference, resources *common.ResourceMap) error {
 	for _, cert := range refs {
+		var mErr error
 		// Verify that the reference has a group and kind that we support
-		if !common.NilOrEqual(cert.Group, "") || !common.NilOrEqual(cert.Kind, common.KindSecret) {
-			return errListenerInvalidCertificateRef_NotSupported
+		if !common.NilOrEqual(cert.Group, "") {
+			mErr = errors.Join(mErr, fmt.Errorf("group is not supported: %q, supported Groups are \"\"", common.DerefStringOr(cert.Group, "")))
```

Comment:
should this be supported Groups are %s, or is the only supported group always just ""

Reply:
for now it's always `""` based on the conditional on the line above that triggers this